rotkehlchen/data/all_assets.json

```diff
@@ -5085,6 +5085,15 @@
         "symbol": "FOTA",
         "type": "ethereum token"
     },
+    "FOX": {
+        "coingecko": "fox-token",
+        "ethereum_address": "0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d",
+        "ethereum_token_decimals": 18,
+        "name": "FOX Token",
+        "started": 1592535600,
```

Comment:
Shouldn't this be `1553639047`?
Seems to have been deployed much earlier: https://etherscan.io/tx/0x590c4ad4f2ea141cffa2dcf10230afc48778c85ce785bcfb96a12fb509dd26ce

Reply:
Yes!  I was working off the earliest datetime seen on CoinGecko, but I see now how to go about this properly.  Thanks!